File: java/program/Clothing.java
// Kelas turunan Clothing
class Clothing extends Product {
    protected String size;
    protected String material;
    protected String gender;

    // Konstruktor
    public Clothing(int idProduct, String name, String brand, double price, String size, String material, String gender) {
        super(idProduct, name, brand, price);
        this.size = size;
        this.material = material;
        this.gender = gender;
    }

    public String getSize() {
        return size;
    }

    public String getMaterial() {
        return material;
    }

    public String getGender() {
        return gender;
    }
}

File: python/program/Clothing.py
from Product import Product

class Clothing(Product):
    def __init__(self, idProduct, name, brand, price, size, material, gender):
        super().__init__(idProduct, name, brand, price)
        self.size = size
        self.material = material
        self.gender = gender

    def getSize(self):
        return self.size

    def getMaterial(self):
        return self.material

    def getGender(self):
        return self.gender
